### ISSUE TITLE
Getting when using GetAtt for Property of type AWS::ElasticLoadBalancing::LoadBalancer

### ISSUE THREAD

**Issue by dthielking** [aws-cloudformation/cfn-python-lint#888]
cfn-lint version: cfn-lint 0.20.1

Hello,
I get following message: 
`E3008 Property "SecurityGroups" has no valid Fn::GetAtt options at Resources/AppLb/Properties/SecurityGroups/0/Fn::GetAtt
template.yaml:87:7`

I'm referencing two SecurityGroups IDs for the LoadBalancer.
```yaml
  # Application Loadbalancer
  AppLbAccessSg:
    Type: AWS::EC2::SecurityGroup
    Properties:
      GroupDescription: Security group for loadbalancer
      VpcId: !ImportValue VPC1-VPC-ID

  AppLbAssocSg:
    Type: AWS::EC2::SecurityGroup
    Properties:
      GroupDescription: This group is used to get allowed by ASG instances
      VpcId: !ImportValue VPC1-VPC-ID
      
  AppLb:
    Type: AWS::ElasticLoadBalancing::LoadBalancer
    Properties:
      ConnectionDrainingPolicy:
        Enabled: true
        Timeout: 60
      HealthCheck:
        HealthyThreshold: '2'
        Interval: '30'
        Target: TCP:8004
        Timeout: '15'
        UnhealthyThreshold: '5'
      Listeners:
        - InstancePort: '8004'
          InstanceProtocol: tcp
          LoadBalancerPort: '8004'
          Protocol: tcp
      LoadBalancerName: !Sub 'lb-${Environment}-app'
      Scheme: internal
      SecurityGroups:
        - !GetAtt AppLbAccessSg.GroupId # This is the position from which I get the messages
        - !GetAtt AppLbAssocSg.GroupId # Same here
      Subnets:
        - !ImportValue VPC1-AZ1Subnet1
        - !ImportValue VPC1-AZ2Subnet1
        - !ImportValue VPC1-AZ3Subnet1
```
Further in my template I am using more !GetAtt function calls and it is working.
Hope this is helpful.

Regards

**Comment by phene** [comment]
You can and should just use `!Ref AppLbAccessSg` instead of `!GetAtt AppLbAccessSg.GroupId` there.
